website/server/libs/bannedWords.js

```diff
@@ -0,0 +1,3 @@
+let bannedWords = ['TEST_PLACEHOLDER_SWEAR_WORD_HERE'];
+
+module.exports = bannedWords;
```

Comment:
With our new system, we should probably not do a direct commit. Also, the commit will still appear in Github. I think the title of this PR should have enough prevention (including the long content warning)